TheWarWithin/DemonHunterVengeance.lua

```diff
@@ -1177,43 +1207,17 @@ spec:RegisterAbilities( {
 
         start = function ()
             applyBuff( "fel_devastation" )
-            removeBuff( "demonsurge_fel_desolation" )
-            if talent.demonic_intensity.enabled then addStack( "demonsurge" ) end
 
-            -- This is likely repeated per tick but it's not worth the CPU overhead to model each tick.
-            if legendary.agony_gaze.enabled and debuff.sinful_brand.up then
-                debuff.sinful_brand.expires = debuff.sinful_brand.expires + 0.75
+            if talent.demonic_intensity.enabled and buff.demonsurge_fel_Desolation.up then
```

Comment:
Big `D`

Reply:
Resolved but not committed?  Just making sure we don't overlook the capitalization error.